### ISSUE TITLE
IA Tracker: 20-10206 FOIA Request

### ISSUE THREAD

**Issue by jennymayoco** [department-of-veterans-affairs/va.gov-team#66281]
### Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

### Links

- [Intake Ticket #65348](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65348)

### Team

- Team Name: Veteran forms team
- Designer: Rachel Shearer
- Product Manager: Tricia Baker
- Slack (_optional_): #veteran-facing-forms

### Timeline
_When does IA review need to be completed by? When is Midpoint Review?_

Midpoint was 9/28. Development for the form started 10/2 and it will be about 3 or weeks before team needs URLs and sitemap placement.

---

### Work Needed (if applicable)

- IA Spec: **Needed**
- Feedback for product team: **Needed**

### Tasks
Check all that apply.

- [X] Create process ticket draft

 ❌  Watch Design Intent - design intent skipped
- [x] Determine IA deliverables
- [x] Collaborate with CAIA
- [x] Ask questions to product team
- [x] Create IA Spec
- [ ] Mikki reviews IA Spec
- [x] Share IA Spec with product team
- [ ] Update Site Map and VA.gov Mega IA sheet

## Acceptance Criteria
- [ ] Feedback is shared with product team in Intake Ticket.

**Comment by jennymayoco** [comment]
Updates:

- Shared entry points with team, but list isn't exhaustive at this moment
- Form is currently in development

**Comment by jennymayoco** [comment]
Update:
- [IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/form-digitization-20-10206.md) has been shared with forms team and Megan
- We don't currently have confirmation on whether Form VA3288 will be fully removed from the database, but connected with Jill just to understand the backend process.

**Comment by jennymayoco** [comment]
Updated IA spec with a new entry point, the Supporting Forms page. This is because we aren't able to add a react widget to the hub page. The Supporting Forms page will act as the static page for the react widget, but the true home is still the Records hub page.

**Comment by mnorthuis** [comment]
@kristinoletmuskat @strelichl This form was soft launched via the supporting forms page so that is where the initial entry point was placed (in order to use the flipper) but that is not the primary, nor the only entry points needed for this. There should have been follow up work to complete these, but looks like that didn't happen and this work was dropped.  

Can you please either use this ticket or open a new ticket to complete the entry points for this form and make sure they are implemented?  Thank you.

**Comment by scjwalker** [comment]
Closing the tracker ticket:
- IA work is done
- Megan will make the two prioritized navigation changes
- Megan will review the needed cross-links, and implement as Content work.

**Comment by megzehn** [comment]
Hi @scjwalker!

Thanks for moving this along, and apologies for my delay. Here are some initial updates on these crosslinks:

- [ ] [Records hub page](https://www.va.gov/records/) - @saratorres2, I added a link to the Records hub page. Could you copy edit the small amount of added text and publish? 
- [X] [Records left nav](https://www.va.gov/records/get-veteran-id-cards/) - added 10.29.2024. @scjwalker let me know if everything looks OK on the left nav placement. It doesn't match the order of the hub page, so we may need to tweak it, but I wanted to align with you on that first. 
- [X] [About VA Form 20-10206](https://www.va.gov/find-forms/about-form-20-10206/)- completed as of 10.17.2024 - @saratorres2, I added some "When to use this form" text. Could you copy edit and publish? 
- [ ] [Privacy Policy](https://www.va.gov/privacy-policy/) - still needed as of 10.17.2024 - I wasn't sure if or how we want to add this one since the primary purpose seems to be concerns around PII collected rather than how to request records. I'll check with Beth or Danielle. 
- [X] [VA claim exam (C&P exam)](https://www.va.gov/disability/va-claim-exam/) - completed as of 10.17.2024
- [X] [Privacy Act Requests](https://department.va.gov/privacy/privacy-act-requests/) - Update on 10.29: The team that manages this page replaced the PDF form link with a link to the About VA Form 20-10206 page (which links to the online version). 
- [ ] [FOIA Requests](https://department.va.gov/foia/foia-requests/) - We don't manage this page, but I'll check with Beth or Danielle. 
- [X] [Supporting Forms page](https://va.gov/supporting-forms-for-claims/) - Complete as of 10.17.2024

**Comment by scjwalker** [comment]
@megzehn thank you for all this careful work! 
- I checked all the links that are in place, in your list above, and navigated to the form successfully from all of them.
- in the Records left nav, could we move 'Request personal records online' down one place, so that it's after 'Get medical records'? Everything else looks great!

![Image](https://github.com/user-attachments/assets/ff045e3c-c992-4a66-9adf-14af3e791212)

**Comment by saratorres2** [comment]
@megzehn @scjwalker Confirming that I published the updates to the [Records hub page](https://www.va.gov/records/) and the [About VA Form 20-10206](https://www.va.gov/find-forms/about-form-20-10206/) page and that the changes are now live!

**Comment by megzehn** [comment]
Thank you @saratorres2!

And thank you @scjwalker for moving this along and reviewing the links! The nav order should be updated when the next content build releases. 

I still need to check with Danielle/Beth about the [Privacy Policy](https://www.va.gov/privacy-policy/) and [FOIA Requests](https://department.va.gov/foia/foia-requests/) pages, but I'll create a new ticket if there's work for those.

**Comment by megzehn** [comment]
Update: I chatted with Danielle and she agreed that we probably don't need to add it to the [Privacy Policy](https://department.va.gov/foia/foia-requests/) page since that's more focused on privacy policy rather than securing records. But she did think we could use it on the FOIA requests page. She's going to look for a contact for us to reach out to.

**Comment by megzehn** [comment]
Re-opening this, because there's one crosslink that got missed -- the [VA claim exam (C&P exam)](https://www.va.gov/disability/va-claim-exam/) page.

I just added it in there. @saratorres2 could you review my changes and publish when you get a minute? Thank you!! And sorry for my oversight on this one! 

Page to review and publish: http://preview-prod.vfs.va.gov/preview?nodeId=914

**Comment by scjwalker** [comment]
All navigation needed for IA requirements is in place! Un-tagging myself.

**Comment by saratorres2** [comment]
@megzehn Changes published. Waiting on live updates.
